packages/@angular/cli/models/webpack-configs/browser.ts

```diff
@@ -80,8 +80,10 @@ export function getBrowserConfig(wco: WebpackConfigOptions) {
   return {
     resolve: {
       mainFields: [
+        'module',
```

Comment:
Something is very broken if `module` is always resolved before the others. This needs to be investigated further.

Reply:
From `@angular/*` packages:

```
  "main": "./bundles/compiler.umd.js",
  "module": "./esm5/compiler.js",
  "es2015": "./esm2015/compiler.js",
```

`es2015` should definitely go first in the `mainFields`, otherwise ES2015 target will use ES5 build of Angular.